internal/distro/fedora/distro.go

```diff
@@ -980,6 +998,24 @@ func newDistro(version int) distro.Distro {
 		},
 		iotRawImgType,
 	)
+	x86_64.addImageTypes(
+		&platform.X86{
+			BIOS: true,
+			BasePlatform: platform.BasePlatform{
+				ImageFormat: platform.FORMAT_RAW,
+			},
+		},
+		minimalrpmImgType,
```

Comment:
This is BIOS (non-UEFI) only.  To enable UEFI for an image, the vendor needs to be set (like you did already for aarch64).  I assume we want this image to support both on x86.